project-api/project-api-test/src/test/java/io/ballerina/projects/test/plugins/CompilerPluginTests.java

```diff
@@ -146,6 +147,20 @@ public void testInBuiltCompilerPlugin() throws IOException {
         Assert.assertTrue(logFileContent.contains(diagnosticResult.errors().stream().findFirst().get().toString()));
     }
 
+    @Test(description = "Test `pkg-analyzer` compiler plugin by checking invalid export modules in `Ballerina.toml`")
```

Comment:
Do we have a test case for valid exports? if not shall we add one to make sure it passes the validation.

Reply:
Added